### ISSUE TITLE
conda install: pip failed, No matching distribution found for codecov==2.1.12

### ISSUE THREAD

**Issue by ShannonTown** [databio/pepatac#241]
Hi, I ran into the following error when trying to install pepatac using conda. It seems that codecov package has been removed from PyPi as deprecated. How to bypass this if I still want to use conda? Thank you!

> Installing pip dependencies: \ Ran pip subprocess with arguments:
> ['/*************/miniconda3/envs/pepatac/bin/python', '-m', 'pip', 'install', '-U', '-r', '/**************/pepatac/condaenv.og390thz.requirements.txt']
> Pip subprocess output:
> Collecting bio==1.3.2
>   Downloading bio-1.3.2-py3-none-any.whl (272 kB)
> Collecting biopython==1.79
>   Downloading biopython-1.79-cp39-cp39-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (2.7 MB)
> Collecting biothings-client==0.2.6
>   Downloading biothings_client-0.2.6-py2.py3-none-any.whl (37 kB)
> Collecting certifi==2021.10.8
>   Using cached certifi-2021.10.8-py2.py3-none-any.whl (149 kB)
> Collecting charset-normalizer==2.0.7
>   Using cached charset_normalizer-2.0.7-py3-none-any.whl (38 kB)
> 
> Pip subprocess error:
> ERROR: Could not find a version that satisfies the requirement codecov==2.1.12 (from versions: 2.1.13)
> ERROR: No matching distribution found for codecov==2.1.12
>                                                                                                                                                                                                    failed
> 
> CondaEnvException: Pip failed

**Comment by ayobi** [comment]
Hi @ShannonTown, you should be able to install pepatac using conda :+1:

**Comment by Ruthwww** [comment]
> Hi @ShannonTown, you should be able to install pepatac using conda now 👍

Could u tell me how to do it? u mean using `conda install` to directly download the PEATAC?

**Comment by ShannonTown** [comment]
@Ruthwww Here is how I was planning to do it:
http://pepatac.databio.org/en/latest/run-conda/

@ayobi Thank you very much! I didn't have a chance to actually try installing pepatac using conda in the end, because I already installed all required packages manually, and passed the "confirm installation" using `./checkinstall`. I was able to run the pipeline, except that the "peak_chromosome_distribution" and "peak_TSS_distribution" plots are empty. Is there a quick solution to this issue without reinstalling pepatac from scratch using conda?
